.cbmc-batch/jobs/aws_hash_table_move/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+#There are no loops in the code under test, so use the biggest possible value
+DEFINES += -DMAX_TABLE_SIZE=SIZE_MAX
```

Comment:
Following the same standard as `aws_string` and `aws_array_list`, wouldn't that be defined in a `Makefile.aws_hash_table` file?

Reply:
Actually, since we're just leaving it unbounded, I'll move that to the c file.